scout/server/blueprints/genes/templates/genes/genes.html

```diff
@@ -15,7 +15,9 @@
 
 <div class="container-float">
     <div class="card panel-default w-100 mt-3 justify-content-center">
-        <div class="panel-heading">Genes</div>
+        <div class="panel-heading">Genes
+          <span style="float:right;">Number of documents: build37->{{nr_genes[0]}} | build38->{{nr_genes[1]}} - Last updated: {{last_updated.strftime('%d-%m-%Y %H:%M:%S')}}</span>
```

Comment:
Good stuff! How about some font styles on this line instead/in addition to the punctuation, or at least avoid the `->`. "Number of documents" is probably a little more under-the-hood than helps the user. 😊 
 
```suggestion
          <span style="float:right;"> {{nr_genes[0]}} GRCh37 genes | {{nr_genes[1]}} GRCh38 genes | Last updated  {{last_updated.strftime('%d-%m-%Y %H:%M:%S')}}</span>
```

Reply:
YES, much better!